ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/execution/BaseJsonRpcProcessor.java

```diff
@@ -46,6 +46,7 @@ public JsonRpcResponse process(
     } catch (final MultiTenancyValidationException e) {
       return new JsonRpcUnauthorizedResponse(id, JsonRpcError.UNAUTHORIZED);
     } catch (final RuntimeException e) {
+      LOG.error("Error processing method: {}", method.getName(), e);
```

Comment:
I think logging it is correct.  I might even go as far as creating a JsonRpcError that contains the stacktrace in its data field.

Reply:
The JsonRpcError is created on the line below (with "internal error" message, but no data), however that is what is returned to the client. I don't think this besu stacktrace is useful on the CL side, better to log it out in besu I think.